store/sqlstore/emoji_store.go

```diff
@@ -157,13 +158,11 @@ func (es SqlEmojiStore) getBy(what string, key interface{}, addToCache bool) (*m
 			`+what+` = :Key
 			AND DeleteAt = 0`, map[string]interface{}{"Key": key})
 	if err != nil {
-		var status int
 		if err == sql.ErrNoRows {
-			status = http.StatusNotFound
-		} else {
-			status = http.StatusInternalServerError
+			return nil, store.NewErrNotFound("Emoji", "")
```

Comment:
Let's add `what+"="+key` as the id in `NewErrNotFound`. 

We can change the signature of `getBy` to have both what and key as strings.